genie-web/src/main/resources/application.yml

```diff
@@ -26,7 +26,7 @@ eureka:
 genie:
   file:
     cache:
-      location: "file:///tmp/genie/cache"
+      location: /tmp/genie/cache
```

Comment:
My concern here is the difference in formatting between this and the jobs directory configuration. The way the jobs dir is used it HAS to have that ```file://``` prefix or some pieces of the code weren't working properly. I believe it was in the bean setup when we try to create the directory if it doesn't exist. I forget exactly just know it's a problem so I would like these to be consistent to avoid problems in user configuration down the road.